content/flux/v0.x/data-types/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Work with Flux data types
+description: >
+  ...
+menu:
+  flux_0_x:
+    name: Work with data types
+weight: 4
+---
+
+A Flux **type** defines the set of values and operations possible on those values.
+Data types are never explicitly declared as part of the Flux syntax except as
+part of a [builtin statement](/flux/v0.x/spec/system-built-ins/).
+Types are inferred from how a value is used.
+
+Flux is a **statically typed**. Variable types are inferred at compile time based
+on how variables are used.
```

Comment:
The verbiage about types not being explicitly declared as part of the syntax is important. In some languages, you have to specifically tell the compiler the type of a value. For example:

```js
/**
 * @param {Date} myDate - The date
 * @param {string} myString - The string
 */
function myFunction(myDate, myString) {
  // ...
}
```

The example above has to explicitly define the type of parameters. In Flux, you don't have to do that because it's statically typed.